toolkit/tools/pkg/imagecustomizerlib/customizeverity_test.go

```diff
@@ -178,7 +178,7 @@ func testCustomizeImageVerityCosiExtractHelper(t *testing.T, testName string, ba
 	// implementation details, and randomness. So, just enforce that the final size is below an arbitary value. Values
 	// were picked by observing values seen during test and adding a good buffer.
 	assert.Greater(t, int64(150*diskutils.MiB), bootStat.Size())
-	assert.Greater(t, int64(650*diskutils.MiB), rootStat.Size())
+	assert.Greater(t, int64(655*diskutils.MiB), rootStat.Size())
```

Comment:
I would use `675` just to ensure there is enough added buffer to prevent this from happening again.